ballerina-command/src/main/resources/cli-help/ballerina-dist.help

```diff
@@ -17,11 +17,13 @@ OPTIONS
 
 
 BALLERINA COMMANDS
-       Here is a list of available sub-commands:
-           list       List installed and available distributions
-           update     Update to the latest patch version of the active distribution
-           pull       Fetch a given distribution and set it as the active version
-           use        Make the specified distribution the active version
-           remove     Delete an installed distribution
-
-       See 'ballerina dist <command> -h' for more information on a specific command.
+       Here is a list of available subcommands:
+
+       update     Update to the latest patch version of the active distribution
+       pull       Fetch a distribution and set it as the active version
+       use        Mark a distribution as the active distribution
+       list       List locally and remotely available distributions
+       remove     Remove distributions in your local environment
+
+
+Use 'ballerina dist <command> -h' for more information on a specific command.
```

Comment:
Should we make this 
```suggestion
Use 'ballerina help dist <command>' for more information on a specific command.
```
to be consistent with https://github.com/ballerina-platform/ballerina-distribution/pull/366/files#diff-e8f479df0634263926d1d97f23b4d46dR7?